test/hardhat/integration/xrp-genesis.test.js

```
@@ -0,0 +1,395 @@
+    );
+    await bootstrapLogic.waitForDeployment();
+    console.log(`Bootstrap contract logic deployed at: ${bootstrapLogic.target}`);
+
+    const ProxyAdmin = await ethers.getContractFactory("ProxyAdmin");
+    const proxyAdmin = await ProxyAdmin.deploy();
+    await proxyAdmin.waitForDeployment();
+    console.log(`ProxyAdmin deployed at: ${proxyAdmin.target}`);
+
+    const initializeArgs = generateBootstrapInitData();
+
+    const Proxy = await ethers.getContractFactory("TransparentUpgradeableProxy");
```

Biome (1.9.4)

[error] Do not shadow the global "Proxy" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)